betfund_bet365/client/client.py

```diff
@@ -0,0 +1,247 @@
+"""
+Betfund Bet365 API Wrapper.
+
+Bet365 Serves as a Client to make requests to Bet365 API
+see for documentation: (https://1394697259.gitbook.io/bet365-api/)
+
+Bet365 Exposes 6 Endpoints:
+    Result ["GET"]
+    InPlay Filter ["GET"]
+    InPlay Odds ["GET"]
+    PreMatch odds ["GET"]
+    InPlay Events ["GET"]
+    Upcoming Events ["GET"]
+
+Responses are parsed into Facade Access objects (Base Bet365Response)
+"""
+from typing import Optional, Union
+from urllib.parse import urljoin
+
+import requests
+
+import betfund_bet365.response as facades
+
+from betfund_bet365.response import Bet365Response
```

Comment:
Need for this import `import betfund_bet365.response as facades`? Also line spacing.

Reply:
IRL discussed - needed for factory